lib/kuiq/model/process.rb

```diff
@@ -0,0 +1,29 @@
+module Kuiq
+  module Model
+    class ProcessSet
+      def initialize
+        @ps = Sidekiq::ProcessSet.new
+      end
+
+      def each
+        @ps.each do |hash|
+          yield Kuiq::Process.new(hash)
+        end
+      end
```

Comment:
`yield` can be quite vague. It would be more understandable if `each` takes an & argument that has a meaningful name that clearly describes what the block does, like &iterator, &processor, &converter, etc...

Example implementation:

```ruby
def each(&iterator)
  @sidekiq_process_set.each do |hash|
    iterator.call(Kuiq::Process.new(hash))
  end
end
```